compiler/src/parsing/ast_mapper.re

```diff
@@ -167,6 +168,29 @@ module D = {
   };
 };
 
+module Exc = {
+  let map = (sub, {ptyexn_constructor: ext, ptyexn_loc: loc}) => {
+    open Except;
+    let cloc = sub.location(sub, loc);
+    let {pext_name: n, pext_kind: k, pext_loc: loc} = ext;
+    let name = map_loc(sub, n);
+    let loc = sub.location(sub, loc);
```

Comment:
@ospencer I know this has already been merged, but why does `cloc` and `loc` use the same `sub.location(sub, loc);` call?

Reply:
@phated `loc` is shadowed here on line 175, but maybe that's not clear 😛